test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3491,6 +3491,102 @@ public async Task MaxOrchestrationAction_MaxReached_OrchestrationFails()
             }
         }
 
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory + "_BVT")]
+        [MemberData(nameof(TestDataGenerator.GetExtendedSessionAndFullFeaturedStorageProviderOptions), MemberType = typeof(TestDataGenerator))]
+        public async Task DedupeStates_NotRunningStates_ThrowsException(bool extendedSessions, string storageProvider)
+        {
+            DurableTaskOptions options = new DurableTaskOptions();
+            options.OverridableExistingInstanceStates = OverridableStates.NonRunningStates;

```

Comment:
This is the default value, right? If so, can we remove this line? That way this test will fail if we accidentally change that default in the future.